Sources/MacroToolkit/NormalizedTypes/NormalizedMemberType.swift

```diff
@@ -0,0 +1,15 @@
+import SwiftSyntax
+
+/// Wraps a member type (e.g. `Array<Int>.Element`).
+public struct NormalizedMemberType: TypeProtocol {
```

Comment:
Add a way to access the root and member types (as normalized types). Same goes for any other normalized types which don't have such properties yet (they should have a simple way to pull out their normalized component parts).

Reply:
Same as https://github.com/stackotter/swift-macro-toolkit/pull/15#discussion_r1684684292 above, `MemberType` does not have any direct way to access such properties.